static/usage.html

```diff
@@ -1021,6 +1022,52 @@ <h2><a href="#app-link-verification">App link verification</a></h2>
                 <a href="https://developer.android.com/training/app-links/verify-site-associations">developer
                 documentation on app link verification</a>.</p>
             </section>
+
+            <section id="carrier-functionality">
+                <h2><a href="#carrier-functionality">Carrier functionality</a></h2>
+
+                <p>GrapheneOS aims to work with all carriers that are officially supported by Google
+                on the stock operating system on Pixel devices. Wi-Fi Calling, VoLTE, Visual Voicemail,
+                MMS, SMS, Calling, 5G (SA and NSA) all are supported however some functionality may 
```

Comment:
A comma should be placed in between "supported" and "however" since this is a very long single sentence.

`supported, however`